src/service/viewport/viewport-binding-natural.js

```diff
@@ -222,7 +222,12 @@ export class ViewportBindingNatural_ {
 
   /** @override */
   getLayoutRect(el, opt_scrollLeft, opt_scrollTop) {
-    const b = el./*OK*/getBoundingClientRect();
+    // Default value for when 'el' is has not been added to the page yet.
+    let b = 0;
+    if (el != undefined && el != null) {
```

Comment:
Two problems with this:

1. Our type checking system already asserts that `el` is not null [1].
2. This does not address the cause of the bug. Calling a function on a null/undefined object yields a different runtime error (try it out yourself).

[1] https://github.com/ampproject/amphtml/blob/4cd12f1f5dc814f3df7cd35d5849c545663cab04/src/service/viewport/viewport-binding-def.js#L167-L176

About Closure Compiler annotations: https://github.com/google/closure-compiler/wiki/Annotating-JavaScript-for-the-Closure-Compiler